src/actions/vstudio/_vstudio.lua

```diff
@@ -338,7 +338,7 @@
 		end
 
 		-- Then the system libraries, which come undecorated
-		local system = config.getlinks(cfg, "system", "fullpath")
+		local system = config.getlinks(cfg, "system", "name")
```

Comment:
hmm... but then we'd have to strip that off again for paths that are already in the libdirs. That was kind of the thing I was trying to achieve... maybe we should cover these cases with a couple more unit-tests...